src/System.Private.CoreLib/shared/System/MemoryExtensions.Fast.cs

```diff
@@ -14,7 +14,7 @@ namespace System
     /// <summary>
     /// Extension methods for Span{T}, Memory{T}, and friends.
     /// </summary>
-    public static partial class MemoryExtensions
+    public static partial class MemoryExtensions // .Fast
```

Comment:
Out of the 600+ partial classes in CoreLib, comment like this is on 3 of them. I do not think we should make it a convention. It just duplicates what you can see from filename. If you would like, you can delete this comment on the 3 partial classes that it is on.